tensorflow/compiler/mlir/lite/BUILD

```diff
@@ -890,6 +890,7 @@ tf_native_cc_binary(
     srcs = ["flatbuffer_to_string.cc"],
     deps = [
         "//tensorflow/lite/schema:schema_fbs_with_reflection",
+        ":tensorflow_lite",
```

Comment:
Why is this needed?

Reply:
This line was added here so that `FlatBufferModel::ByteSwapSerializedModel()` could be invoked in `tensorflow/compiler/mlir/lite/flatbuffer_to_string.cc`. 

The conversion could fix a series of `//tensorflow/compiler/mlir/lite/tests/mlir2flatbuffer` test cases in `compiler` module on BE machines.

Reply:
Yes, I'll improve the code to use `model_builder` only. Thanks!

Reply:
Done.